src/plugins/intel_npu/src/backend/src/zero_init.cpp

```diff
@@ -201,6 +200,7 @@ ZeroInitStructsHolder::ZeroInitStructsHolder() : log("NPUZeroInitStructsHolder",
     // Get our target device
     zeroUtils::throwOnFail("zeDeviceGet", zeDeviceGet(driver_handle, &device_count, &device_handle));
 
+    // Create context from backend to be shared with compiler
```

Comment:
context is used in the backend as well :)

Reply:
i have update this commit,  context created is share between compiler and backend